Index: trunk/mingw-w64-crt/Makefile.in
===================================================================
--- trunk/mingw-w64-crt/Makefile.in	(revision 5198)
+++ trunk/mingw-w64-crt/Makefile.in	(revision 5199)
@@ -3527,7 +3527,7 @@
 #########
 # Compiling 32-bit runtime
 #
-@LIB32_TRUE@lib32dir = $(prefix)/$(host)/@LIB32SUFFIXDIR@
+@LIB32_TRUE@lib32dir = $(prefix)/@LIB32SUFFIXDIR@
 @LIB32_TRUE@crt32dir = $(lib32dir)
 @LIB32_TRUE@crt32_DATA = \
 @LIB32_TRUE@  lib32/crt1.o    lib32/crt2.o     lib32/crt1u.o   lib32/crt2u.o    lib32/dllcrt1.o    lib32/dllcrt2.o \
@@ -3722,7 +3722,7 @@
 #########################################################################
 # Compiling 64-bit runtime
 # #######################################################################
-@LIB64_TRUE@lib64dir = $(prefix)/$(host)/@LIB64SUFFIXDIR@
+@LIB64_TRUE@lib64dir = $(prefix)/@LIB64SUFFIXDIR@
 @LIB64_TRUE@crt64dir = $(lib64dir)
 @LIB64_TRUE@crt64_DATA = \
 @LIB64_TRUE@  lib64/crt1.o    lib64/crt2.o     lib64/crt1u.o   lib64/crt2u.o   lib64/dllcrt1.o lib64/dllcrt2.o \
@@ -4088,7 +4088,7 @@
 
 # Don't compile these, but install as is:
 #
-libsrcdir = $(prefix)/$(host)/libsrc
+libsrcdir = $(prefix)/libsrc
 libsrc_DATA = \
   libsrc/agtctl_i.c   libsrc/agtsvr_i.c     libsrc/cdoex_i.c    libsrc/cdoexm_i.c     \
   libsrc/cdosys_i.c   libsrc/emostore_i.c   libsrc/iisext_i.c   libsrc/mtsadmin_i.c   \
Index: trunk/mingw-w64-crt/Makefile.am
===================================================================
--- trunk/mingw-w64-crt/Makefile.am	(revision 5198)
+++ trunk/mingw-w64-crt/Makefile.am	(revision 5199)
@@ -288,7 +288,7 @@
 # Compiling 32-bit runtime
 #
 
-lib32dir=$(prefix)/$(host)/@LIB32SUFFIXDIR@
+lib32dir=$(prefix)/@LIB32SUFFIXDIR@
 crt32dir=$(lib32dir)
 
 crt32_DATA = \
@@ -585,7 +585,7 @@
 # Compiling 64-bit runtime
 # #######################################################################
 
-lib64dir=$(prefix)/$(host)/@LIB64SUFFIXDIR@
+lib64dir=$(prefix)/@LIB64SUFFIXDIR@
 crt64dir=$(lib64dir)
 
 crt64_DATA = \
@@ -1050,7 +1050,7 @@
 
 # Don't compile these, but install as is:
 #
-libsrcdir=$(prefix)/$(host)/libsrc
+libsrcdir=$(prefix)/libsrc
 libsrc_DATA = \
   libsrc/agtctl_i.c   libsrc/agtsvr_i.c     libsrc/cdoex_i.c    libsrc/cdoexm_i.c     \
   libsrc/cdosys_i.c   libsrc/emostore_i.c   libsrc/iisext_i.c   libsrc/mtsadmin_i.c   \
