diff -rupN --no-dereference mingw-w64-v9.0.0/mingw-w64-crt/intrincs/rdtsc.c mingw-w64-v9.0.0-new/mingw-w64-crt/intrincs/rdtsc.c
--- mingw-w64-v9.0.0/mingw-w64-crt/intrincs/rdtsc.c	2021-05-22 14:34:25.000000000 +0200
+++ mingw-w64-v9.0.0-new/mingw-w64-crt/intrincs/rdtsc.c	2022-06-03 15:11:49.341407410 +0200
@@ -15,7 +15,7 @@
 #endif
 
 #if !__has_builtin(__rdtsc)
-unsigned __int64 __rdtsc(void)
+unsigned __int64 __rdtsc()
 {
 #ifdef _WIN64
       unsigned __int64 val1, val2;
