Index: trunk/mingw-w64-crt/configure
===================================================================
--- trunk/mingw-w64-crt/configure	(revision 5196)
+++ trunk/mingw-w64-crt/configure	(revision 5197)
@@ -682,9 +682,9 @@
 CFLAGS
 CC
 SED
-TARGET_SYSTEM_ROOT
 WITHSYSROOT_FALSE
 WITHSYSROOT_TRUE
+TARGET_SYSTEM_ROOT
 host_os
 host_vendor
 host_cpu
@@ -1430,7 +1430,8 @@
 Optional Packages:
   --with-PACKAGE[=ARG]    use PACKAGE [ARG=yes]
   --without-PACKAGE       do not use PACKAGE (same as --with-PACKAGE=no)
-  --with-sysroot=DIR      Search for include within DIR/mingw.
+  --with-sysroot=DIR      Search for headers within DIR/include (default:
+                          prefix/include)
 
 Some influential environment variables:
   CC          C compiler command
@@ -2775,46 +2776,28 @@
 
 # Check whether --with-sysroot was given.
 if test "${with_sysroot+set}" = set; then :
-  withval=$with_sysroot; case $with_sysroot in #(
+  withval=$with_sysroot;
+else
+  with_sysroot=yes
+fi
+
+case $with_sysroot in #(
   no) :
-    { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; } ;; #(
+     ;; #(
   yes) :
-    { $as_echo "$as_me:${as_lineno-$LINENO}: result: none specified" >&5
-$as_echo "none specified" >&6; }
-      as_fn_error $? "Please specify a directory to the --with-sysroot=DIR option" "$LINENO" 5
-     ;; #(
+    with_sysroot=$prefix/include ;; #(
   *) :
-
-      { $as_echo "$as_me:${as_lineno-$LINENO}: result: $with_sysroot" >&5
-$as_echo "$with_sysroot" >&6; }
-      TARGET_SYSTEM_ROOT=$with_sysroot
-      { $as_echo "$as_me:${as_lineno-$LINENO}: checking if sysroot contains an include directory" >&5
-$as_echo_n "checking if sysroot contains an include directory... " >&6; }
-      if test -d $with_sysroot/$host/include; then :
-  { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-else
-
-          { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-          as_fn_error $? "Sysroot directory does not contain $host/include" "$LINENO" 5
-          with_sysroot=no
-
-fi
      ;;
 esac
+{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $with_sysroot" >&5
+$as_echo "$with_sysroot" >&6; }
+if test "x$with_sysroot" = xno; then :
 
 else
-
-    { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-    TARGET_SYSTEM_ROOT=
-    with_sysroot=no
-
+  TARGET_SYSTEM_ROOT=$with_sysroot
 fi
 
- if test $with_sysroot != no; then
+ if ${TARGET_SYSTEM_ROOT+:} false; then
   WITHSYSROOT_TRUE=
   WITHSYSROOT_FALSE='#'
 else
Index: trunk/mingw-w64-crt/Makefile.in
===================================================================
--- trunk/mingw-w64-crt/Makefile.in	(revision 5196)
+++ trunk/mingw-w64-crt/Makefile.in	(revision 5197)
@@ -3267,7 +3267,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 @WITHSYSROOT_FALSE@sysincludes = 
-@WITHSYSROOT_TRUE@sysincludes = -I@TARGET_SYSTEM_ROOT@/$(host)/include
+@WITHSYSROOT_TRUE@sysincludes = -I@TARGET_SYSTEM_ROOT@/include
 @WITHSYSROOT_FALSE@withsys = 
 @WITHSYSROOT_TRUE@withsys = --with-sysroot=@TARGET_SYSTEM_ROOT@
 @UNDERSCORES_FALSE@cflags_underscores = -fno-leading-underscore
Index: trunk/mingw-w64-crt/configure.ac
===================================================================
--- trunk/mingw-w64-crt/configure.ac	(revision 5196)
+++ trunk/mingw-w64-crt/configure.ac	(revision 5197)
@@ -16,32 +16,19 @@
 AC_MSG_CHECKING([for sysroot])
 AC_ARG_WITH([sysroot],
   [AS_HELP_STRING([--with-sysroot=DIR],
-    [Search for include within DIR/mingw.])],
-  [AS_CASE([$with_sysroot],
-    [no],[AC_MSG_RESULT([no])],
-    [yes],[AC_MSG_RESULT([none specified])
-      AC_MSG_ERROR([Please specify a directory to the --with-sysroot=DIR option])
-    ],[
-      AC_MSG_RESULT([$with_sysroot])
-      TARGET_SYSTEM_ROOT=$with_sysroot
-      AC_MSG_CHECKING([if sysroot contains an include directory])
-      AS_IF([test -d $with_sysroot/$host/include],
-        [AC_MSG_RESULT([yes])],
-        [
-          AC_MSG_RESULT([no])
-          AC_MSG_ERROR([Sysroot directory does not contain $host/include])
-          with_sysroot=no
-        ])
-    ])
-  ],
-  [
-    AC_MSG_RESULT([no])
-    TARGET_SYSTEM_ROOT=
-    with_sysroot=no
-  ])
-AM_CONDITIONAL([WITHSYSROOT],[test $with_sysroot != no])
+    [Search for headers within DIR/include (default: prefix/include)])],
+  [],
+  [AS_VAR_SET([with_sysroot],[yes])])
+AS_CASE([$with_sysroot],
+  [no],[],
+  [yes],[AS_VAR_SET([with_sysroot],[$prefix/include])],
+  [])
+AC_MSG_RESULT([$with_sysroot])
+AS_VAR_IF([with_sysroot],[no],[],[AS_VAR_SET([TARGET_SYSTEM_ROOT],[$with_sysroot])])
 AC_SUBST([TARGET_SYSTEM_ROOT])
+AM_CONDITIONAL([WITHSYSROOT],[AS_VAR_TEST_SET([TARGET_SYSTEM_ROOT])])
 
+
 # Checks for programs.
 # Checks for headers, libs, etc need to utilize the sysroot... no idea how
 AC_NO_EXECUTABLES
Index: trunk/mingw-w64-crt/Makefile.am
===================================================================
--- trunk/mingw-w64-crt/Makefile.am	(revision 5196)
+++ trunk/mingw-w64-crt/Makefile.am	(revision 5197)
@@ -13,7 +13,7 @@
 #AUTOMAKE_OPTIONS = color-tests
 
 if WITHSYSROOT
-  sysincludes=-I@TARGET_SYSTEM_ROOT@/$(host)/include
+  sysincludes=-I@TARGET_SYSTEM_ROOT@/include
   withsys=--with-sysroot=@TARGET_SYSTEM_ROOT@
 else
   sysincludes=
