--- mingw-w64-crt/Makefile.am.ilockcxch_regression	2013-06-14 22:43:42.000000000 +0200
+++ mingw-w64-crt/Makefile.am	2013-06-15 00:04:25.812271438 +0200
@@ -158,7 +158,7 @@
   secapi/wmemcpy_s.c
 
 
-src_libmingwex=\
+src_libmingwex=intrincs/ilockcxch.c intrincs/ilockcxch2.c\
   crt/dllentry.c        crt/dllmain.c \
   \
   complex/cabs.c     complex/cabsf.c    complex/cabsl.c   complex/cacos.c    complex/cacosf.c   complex/cacosl.c \
@@ -263,7 +263,6 @@
   intrincs/bittest.c        intrincs/bittestc.c       intrincs/bittestci.c       intrincs/bittestr.c      intrincs/bittestri.c      \
   intrincs/bittests.c       intrincs/bittestsi.c      intrincs/cpuid.c           intrincs/currentfiber.c  intrincs/currentteb.c     \
   intrincs/fiberdata.c      intrincs/ilockadd.c       intrincs/ilockand.c        intrincs/ilockand64.c \
-  intrincs/ilockcxch.c      intrincs/ilockcxch2.c \
   intrincs/ilockcxch16.c    intrincs/ilockcxch64.c    intrincs/ilockcxchptr.c    intrincs/ilockdec.c      intrincs/ilockdec16.c     \
   intrincs/ilockdec64.c     intrincs/ilockexch.c      intrincs/ilockexch64.c     intrincs/ilockexchadd.c  intrincs/ilockexchadd64.c \
   intrincs/ilockexchptr.c   intrincs/ilockinc.c       intrincs/ilockinc16.c      intrincs/ilockinc64.c    intrincs/ilockor.c        \
